src/text/line.rs

```diff
@@ -100,14 +100,22 @@ impl<'a> Line<'a> {
 }
 
 impl<'a> From<String> for Line<'a> {
+    #[inline]
```

Comment:
I make it a habit of adding `#[inline]` to trivial functions like this, to ensure they are inlined even across crate boundaries.

Reply:
I'd argue for consistency sake if there's no specific performance issue being addressed by inlining these methods we should probably remove the macros. This can always be something we readress crate wide in the future.